web/src/message_list_view.js

```diff
@@ -287,10 +287,14 @@ export class MessageListView {
         if (last_edit_timestamp !== undefined) {
             const last_edit_time = new Date(last_edit_timestamp * 1000);
             const today = new Date();
+            let date = timerender.render_date(last_edit_time, today)[0].textContent;
+            if (date && !/\d/.test(date)) {
```

Comment:
Cool. Let's have you add a comment here explaining what that if condition is checking for, and why. It would also be good for someone, maybe @amanagr, to check this logic and make sure there aren't any edge cases being overlooked here.

Reply:
Added the comment

Reply:
LGTM